### DISCUSSION TITLE
New error occured.

### DISCUSSION THREAD

**Discussion by Faiz-Liners** [lyc8503/onedrive-cf-index-ng#35] [Q&A]
I couldn't figure out a solution. Hope you might be able to repair the files.
I think cloudflare might update something in their system so the configuration is not compatible.

Error for first launch before :

Error: Could not access built-in Node.js modules. Please make sure that your Cloudflare Pages project has the 'nodejs_compat' compatibility flag set.

Error occured now :

21:32:03.056 | ✘ [ERROR] Could not resolve "async_hooks"
-- | --
21:32:03.057 |  
21:32:03.057 | <stdin>:222:680:
21:32:03.057 | 222 │ ...:()=>N});var R=A(require("async_hooks")),S="@next/request-cont...
21:32:03.057 | ╵                             ~~~~~~~~~~~~~
21:32:03.057 |  
21:32:03.057 | The package "async_hooks" wasn't found on the file system but is built into node. Are you trying to bundle for node? You can use "platform: 'node'" to do that, which will remove this error.
21:32:03.058 |  
21:32:03.062 | /opt/buildhome/repo/node_modules/.pnpm/esbuild@0.15.18/node_modules/esbuild/lib/main.js:1575
21:32:03.063 | let error = new Error(`${text}${summary}`);

There's a bit more lines below this error. I only copy partial of it.
Hope you could try and fix it. Thanks in advance.

**Comment by lyc8503** [comment]
Hi, this is a problem introduced in some upstream changes. I have upgraded accordingly to fix it. Please refer to https://github.com/lyc8503/onedrive-cf-index-ng/issues/34.

**Reply by Faiz-Liners** [reply]
Thanks for the quick update!
Works great 😊
